projects/storefrontapp-e2e-cypress/cypress/helpers/customer-ticketing/customer-ticketing.ts

```diff
@@ -0,0 +1,202 @@
+import { clickHamburger } from '../checkout-flow';
+import { waitForPage, visitHomePage } from '../checkout-flow';
+import { loginRegisteredUser as login } from "../cart";
+
+const HTTP_STATUS_OK = 200;
+const FIRST_ROW = 1;
+const SUBJECT_COLUMN = 1;
+const CATEGORY_COLUMN = 2;
+const STATUS_COLUMN = 5;
+
+const CUSTOMER_SUPPORT_MENU_OPTION_INDEX = 14;
+const FIRST_TICKET_COLUMN_INDEX = 1;
+const FIRST_TICKET_ROW_INDEX = 1;
+
+export enum TestCategory {
+  enquiry = "Enquiry",
+  complaint = "Complaint",
+  problem = "Problem",
+}
+
+export interface TestTicketDetails {
+  subject: string;
+  message: string;
+  category: TestCategory;
+  associatedTo?: string;
+  filename?: string;
+}
+
+export function loginRegisteredUser() {
+  login();
+}
+
+export function clickMyAccountMenuOption(){
+  cy.visit('/');
+  cy.get('#cx-header', { timeout: 10000 }).should('be.visible');
```

Comment:
I think the default timeout inside the project is 5 seconds, I might be mistaken

Reply:
I think this is because there's a mistake in the name of the file currently in the epic and once the create ticket e2e is done, these duplicate changes should be gone. Apologies for this

Reply:
The code is now updated of with only new changes